core/src/mindustry/content/UnitTypes.java

```diff
@@ -876,7 +880,8 @@ public void load(){
             engineOffset = 5.5f;
             range = 140f;
 
-            weapons.add(new Weapon(){{
+            weapons.add(
+            new Weapon(){{
```

Comment:
and here

Reply:
All the other weapons are like this, so I made these 3 like the others; make formatting consistent.